catalogue/webapp/components/SearchForm/SearchForm.js

```diff
@@ -57,7 +65,23 @@ const SearchForm = ({ ariaDescribedBy, compact }: Props) => {
     if (query !== inputQuery) {
       setInputQuery(query);
     }
-  }, [query]);
+
+    if (_dateFrom !== inputDateFrom) {
+      setInputDateFrom(_dateFrom);
+    }
+
+    if (_dateTo !== inputDateTo) {
+      setInputDateTo(_dateTo);
+    }
+  }, [query, _dateFrom, _dateTo]);
+
+  function updateDateFrom(event) {
```

Comment:
Is there a reason to abstract these out and not just do it on the element?

Reply:
Probably not anymore, no – thanks. Will update.